app/scripts/options.js

```diff
@@ -272,3 +272,11 @@ app.controller('ShortkeysOptionsCtrl', ['$scope', function ($scope) {
     $scope.$apply()
   })
 }])
+
+try {
+  browser.runtime.getBrowserInfo().then((result) => {
```

Comment:
instead of getting the browser type in runtime - I would prefer to use process.env.VENDOR === 'firefox' as described here: https://github.com/HaNdTriX/generator-chrome-extension-kickstart/blob/HEAD/DOCUMENTATION.md